Update from auto-update/renku-graph-2.32.0 (#3051)
SwissDataScienceCenter/renku: CHANGELOG.rst

.. _changelog:

0.25.4
------

Renku ``0.25.4`` brings bug fixes for KG.

Internal Changes
~~~~~~~~~~~~~~~~

**Bug Fixes**

> - **KG**: issue with synchronizing users that are authorized to see project data in the Triples Store
> - **KG**: the Delete Project API to work when there is no data for the project in the Triples Store
> - **KG**: the Lineage API to work in cases of implicit parameters, inputs and outputs

Reviewer: ```suggestion
- **KG**: resolve issues with synchronization of project user access authorization data
- **KG**: ensure that the Delete Project API works when there is no data for the project in the Triples Store
- **KG**: ensure that the Lineage API works in cases of implicit parameters, inputs and outputs
```
